stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/images/StreamImageLoader.kt

```diff
@@ -14,6 +14,8 @@ public sealed interface StreamImageLoader {
         public fun instance(): StreamImageLoader = CoilStreamImageLoader
     }
 
+    public var imageRequestHeaders: Map<String, String>
```

Comment:
This approach of having a `Map` makes the set of headers included static. Is that the requirement?

Otherwise we could have some `HeaderProvider` class that behaves similarly to `UrlSigner` that we keep invoking to get this data every time we load an image.

Reply:
That might be a good idea. Let me change that